index.rst

```diff
@@ -3,40 +3,65 @@ OpenWISP Documentation
 
 **Need a quick overview?** `Try the OpenWISP Demo <https://openwisp.org/demo.html>`_.
 
-Contents:
+.. ifconfig:: version == '22.05'
 
-.. toctree::
-   :maxdepth: 2
+   .. toctree::
+      :maxdepth: 1
 
-   user/quickstart
-   user/configure-device
-   user/monitoring
-   user/templates
-   user/configuration-variables
-   user/vpn
-   user/wireguard
-   user/zerotier
-   user/configure-push-updates
-   user/send-commands
-   user/subnet-division-rules
-   user/firmware-upgrades
-   user/network-topology
-   user/radius
-   user/wifi-login-pages
-   user/django-settings
-   user/usage-metric-collection
-   tutorials/demo
-   tutorials/wifi-access-point.rst
-   tutorials/hotspot
-   tutorials/mesh
-   general/architecture
-   general/values
-   general/help-us
-   general/press
-   general/code-of-conduct
-   developer/contributing
-   developer/hacking-openwisp-python-django
-   developer/google-summer-of-code
-   developer/gsoc-ideas-2024
-   developer/google-code-in
-   developer/hacktoberfest
+      user/quickstart
+      user/configure-device
+      user/monitoring
+      user/templates
+      user/configuration-variables
+      user/vpn
+      user/wireguard
+      user/zerotier
+      user/configure-push-updates
+      user/send-commands
+      user/subnet-division-rules
+      user/firmware-upgrades
+      user/network-topology
+      user/radius
+      user/wifi-login-pages
+      user/django-settings
+      user/usage-metric-collection
+      tutorials/demo
+      tutorials/wifi-access-point.rst
+      tutorials/hotspot
+      tutorials/mesh
+      general/architecture
+      general/values
+      general/help-us
+      general/press
+      general/code-of-conduct
+      developer/contributing
+      developer/hacking-openwisp-python-django
+      developer/google-summer-of-code
+      developer/gsoc-ideas-2024
+      developer/google-code-in
+      developer/hacktoberfest
+
+.. ifconfig:: version != '22.05'
+
+   .. toctree::
+      :maxdepth: 1
+      :glob:
+
+      user/quickstart
+      modules/*/docs/*
```

Comment:
maybe we need to name the folders with numbers to order them as we want? Would that impact the URLs?

Reply:
Yes, that would change the URLs.